README.md

```diff
@@ -55,32 +61,57 @@ After you have installed GOV.UK Frontend the `@govuk-frontend` package will appe
 ```CSS
 @import "@govuk-frontend/button/button";
 ```
 
+You should place the above code before your other Sass rules, or Sass imports, if you want to override
+GOV.UK Frontend with your own styles.
 
-To resolve your `@import` declarations you should add `node_modules` to your [Sass include paths](https://github.com/sass/node-sass#includepaths) or [assets paths](http://guides.rubyonrails.org/asset_pipeline.html#search-paths) in Ruby.
+In order to resolve the above `@import` paths(s) in your build, you should add `node_modules` to your [Sass include paths](https://github.com/sass/node-sass#includepaths) (in Ruby, they should be added to [assets paths](http://guides.rubyonrails.org/asset_pipeline.html#search-paths)).
+
+For example, if your project uses Gulp, you would add the Sass include paths to your Gulp configuration file (for example `gulpfile.js`) with [gulp-sass](https://www.npmjs.com/package/gulp-sass). Below is an example:
 
-Below is a code sample you can add to your gulp configuration file using the gulp-sass package:
 ```JS
-.pipe(sass({ includePaths: 'node_modules/' }))
+gulp.task('sass', function () {
+  return gulp.src('./sass/**/*.scss')
+    .pipe(sass({
+      includePaths: 'node_modules/@govuk-frontend'
+     }))
+    .pipe(gulp.dest('./css'));
+});
+
 ```
 
-To import images, configure your application to reference or copy the icons assets.
+Note: If you compile Sass to CSS in your project, your build tasks will already include something similar to the above task - in
+that case, you will just need to include add `includePaths` to it.
+
+### Import images and icons
+
+In order to import GOV.UK Frontend images and icons to your project, you should configure your application to reference or copy the relevant GOV.UK Frontend assets.
 
-Below is a code sample you can add to your [express.js](https://expressjs.com/) configuration:
+Recommended solution:
+
+Make `/node_modules/@govuk-frontend/icons` available to your project by routing requests for your images folder there. For example, if your project uses [express.js](https://expressjs.com/), below is a code sample you could add to your configuration:
 ```JS
-app.use('/public', express.static(path.join(__dirname, '/node_modules/@govuk-frontend/icons')))
+app.use('/icons', express.static(path.join(__dirname, '/node_modules/@govuk-frontend/icons')))
 ```
+Alternative solution:
+
+1. Manually copy the images from `/node_modules/@govuk-frontend/icons` into a public facing directory in your project. Ideally copying the files to your project should be an automated task or part of your build pipeline to ensure that the GOV.UK Frontend images and icons stay up-to-date.
+
+2. Set `$global-images` variable in your project Sass file to point to the images folder in your project. Make sure you do this before you import `@govuk-frontend` into your project - see "Import styles" above. (`$global-images` is defined by default in `/node_modules/frontend/global/settings/_paths.scss`.)
```

Comment:
This is true, regardless of using the recommended or the alternative solution. So we can have it as a note for both solutions. 'Import styles' should be a link.

Reply:
Done!